packages/devtools/src/App-2.tsx

```diff
@@ -0,0 +1,104 @@
+import { FC, useEffect, useState } from 'react';
+import { BrowserRouter, Navigate, Route, Routes } from 'react-router';
+import * as icons from '@fluentui/react-icons';
+import { mergeClasses } from '@fluentui/react-components';
+
+import { ConsoleLogger } from '@teams.sdk/common/logging';
+
+import { SocketClient } from './socket-client';
+import { ActivityContext, ChatContext, useActivityStore, useChatStore } from './Stores';
+
+import ChatPane from './components/ChatPane/ChatPane';
+import useGlobalStyles from './useGlobalStyles';
+import Cards from './Screens/Cards';
+import Activities from './Screens/Activities';
+import Logs from './Screens/Logs';
+import DevtoolsLandmark from './Components/DevtoolsLandmark/DevtoolsLandmark';
+import TopNavButton from './Components/TopNavButton/TopNavButton';
+import './App.css';
+
+const socket = new SocketClient();
+const log = new ConsoleLogger('devtools');
+
+const App: FC = () => {
```

Comment:
should this be renamed to `App.tsx` and replace the old one?

Reply:
Yep, will do that eventually, I just wanna make sure I combine the two files correctly before deleting the original App.tsx